eva/udfs/contrib/decorators/ish/io_descriptors/data_types/type_pandas_dataframe.py

```diff
@@ -0,0 +1,41 @@
+# coding=utf-8
+# Copyright 2018-2022 EVA
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pandas as pd
+
+from eva.udfs.contrib.decorators.ish.io_descriptors.eva_arguments import EvaArgument
+
+
+class PandasDataframe(EvaArgument):
```

Comment:
We probably need to discuss how to support pandas dataframe as input with multiple columns. In that case it won't be as straightforward as a tensor or numpy array and cannot extend the EVAArgument class.